### PR TITLE
HSEARCH-3099 follow-up: fix tests and documentation for "indexNullAs" on GeoPoint fields in ES5.6

### PR DESCRIPTION
**Pull request by yrodiere**
https://hibernate.atlassian.net//browse/HSEARCH-3099

Full tests here: http://ci.hibernate.org/job/hibernate-search/job/PR-1961/9/

**Please wait for the result of this build before merging**.

### CODE REVIEW

File: integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/testsupport/types/expectations/IndexNullAsMatchPredicateExpectactions.java [hunk 1]
```diff
@@ -6,12 +6,12 @@
  */
 package org.hibernate.search.integrationtest.backend.tck.testsupport.types.expectations;
 
-public class IndexNullAsExpectactions<F> {
+public class IndexNullAsMatchPredicateExpectactions<F> {
```
[the remark this answers is not in the document]
Comment (yrodiere) [reply integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/testsupport/types/expectations/IndexNullAsMatchPredicateExpectactions.java]: The values may not be, but the expectations are. You're testing indexNullAs with a match predicate, which means these expectations won't hold for a `geo_point` field where the match predicate isn't supported, for example.

[the remark this answers is not in the document]
Comment (fax4ever) [reply integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/testsupport/types/expectations/IndexNullAsMatchPredicateExpectactions.java]: I think that these expectations could hold for a `geo_point` too. But I don't that it is very critical :P So we can keep the change here.

**Comment by fax4ever** [comment]
@yrodiere +1 to merge if it passes the CI

**Comment by yrodiere** [comment]
Merged, thanks

**Pull request merged**